src/components/Dialogs/TransactionDialog/TransactionDialog.tsx

```diff
@@ -2,12 +2,25 @@ import React from 'react'
 import ActionDialog, { ActionDialogProps } from '../ActionDialog/ActionDialog'
 import { TextContainer, StyledTransactionIllustration, Spinner } from './TransactionDialog.style'
 import { StyledTitleText, StyledDescriptionText } from '../MessageDialog/MessageDialog.style'
+import { ExtrinsicStatus } from '@/joystream-lib'
 
-export type TransactionDialogProps = ActionDialogProps
+export type TransactionDialogProps = Omit<
+  ActionDialogProps,
+  'additionalActionsNode' | 'primaryButtonText' | 'onPrimaryButtonClick' | 'secondaryButtonText' | 'showDialog'
+> & {
+  onCancel?: () => void
+  status: ExtrinsicStatus | null
+}
```

Comment:
```suggestion
export type TransactionDialogProps = Pick<ActionDialogProps, 'warning' | 'error'> & {
  onCancel?: () => void
  status: ExtrinsicStatus | null
}
```
I think it should be the same, and is much more readable

Reply:
`Pick` may be a better choice, but we may need some other props. Will take a look

Reply:
Done